api/v1/mongodbcommunity_types.go

```diff
@@ -51,6 +51,10 @@ type MongoDBCommunitySpec struct {
 	// Version defines which version of MongoDB will be used
 	Version string `json:"version"`
 
+	// Arbiters is the number arbiters (each counted as a member) in the replica set
```

Comment:
```suggestion
	// Arbiters is the number of arbiters (each counted as a member) in the replica set
```

Also, I'm not sure `(each counted as a member)` is really clear, but I defer to native speakers for this